packages/vuepress/vuepress-plugin-apidocs/components/PropertyList.vue

```diff
@@ -25,18 +28,35 @@
 import AvailabilityInfo from './AvailabilityInfo'
 import DeprecationAlert from './DeprecationAlert'
 import PropertySignature from './PropertySignature'
+import ValidConstants from './ValidConstants'
 
 export default {
   components: {
     AvailabilityInfo,
     DeprecationAlert,
-    PropertySignature
+    PropertySignature,
+    ValidConstants
   },
   props: {
     properties: {
       type: Array,
       default: () => []
     }
+  },
+  methods: {
+    normalizeType (type) {
+      switch (typeof type) {
+        case 'undefined':
+          return 'undefined'
+        case 'boolean':
+        case 'number':
+          return type.toString()
+        case 'string':
+          return type.replace(/^<|>$/g, '')
```

Comment:
I think this case is intended to be our special "link" format to point to another constant. Can we magically link that up here (You appear to have added that in to our support somewhere already for summaries/descriptions)?

Reply:
Nevermind, looks like this already does work!